lib/hex_web/web/templates/docs/mirrors.html.eex

```diff
@@ -15,7 +15,13 @@
 </p>
 
 <p>
-  <strong>Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+  <strong>Mix Example</strong>: <code>$ mix hex.config cdn_url https://s3.amazonaws.com/s3.hex.pm</code>
+</p>
+<p>
+  <strong>Rebar3 Example</strong>: <code>{rebar_packages_cdn, "https://s3.amazonaws.com/s3.hex.pm"}.</code>
+</p>
```

Comment:
Ok, yea, just realized setting the `cdn_url` through `rebar3 hex config` doesn't really matter, it isn't used by rebar3 at all.